Support a new checksum type for "CRC64NVME"

mozawa opened the pull request:
- Support a new checksum algorithm for "`CRC64NVME`" and a checksum type for "`COMPOSITE`|`FULL_OBJECT`"
- Switch to use APIs in awscrt package to compute CRC* checksum algorithm

mozawa committed:
Support a new checksum type for "CRC64NVME"

tpodowd reviewed (commented):
These look ok to me. Do we have to update the requirements.txt file?

mozawa committed:
add awscrt >= 0.23.8 in requirements.txt

mozawa commented:
> These look ok to me. Do we have to update the requirements.txt file?

Updated [here](https://github.com/cloudian/boto/pull/53/commits/dac4738a15b7967282ebf7acfd1760116d30210a) and removed it from requirements.txt of toms3 [here](https://github.com/cloudian/toms3/pull/73/commits/8bc15049032cf1c82c4df8aed3757efae72462bd)

tpodowd reviewed (approved):
Looks good to me. Thanks for fixing the requirements.

mozawa committed:
use install_requires rather than requirements.txt to install awscrt

tpodowd reviewed (approved):
Using setup.py might just be the way.
